doc_source/deleting-vaults-cli.md

```diff
@@ -112,9 +112,10 @@ S3 Glacier deletes a vault only if there are no archives in the vault as of the
    ```
 
 1. Use the `delete-archive` command to delete each archive from a vault until none remain\.
+The archiveid can start with a "-" this confuses the argument parser if the normal form of --archive-id "-asdfasdf" is used.  using the form --archive-id="-asdfasdf" seems to work reliably
```

Comment:
This sentence (115) is not written in the same style and voice as the rest of this page. If we remove this we can merge the request.